sdk/iot/Azure.IoT.Hub.Service/samples/IotHubClientSamples/BulkDeviceIdentityLifecycleSample.cs

```diff
@@ -5,9 +5,9 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using Azure.Iot.Hub.Service.Models;
+using Azure.IoT.Hub.Service.Models;
```

Comment:
Is it all find and replace?

Reply:
it's all find and replace and fixing some warnings.